src/main/java/com/idrsolutions/microservice/BuildVuServlet.java

```diff
@@ -22,6 +22,7 @@
 
 import com.idrsolutions.microservice.utils.SettingsValidator;
 import com.idrsolutions.microservice.utils.ZipHelper;
+import com.sun.tools.sjavac.Log;
```

Comment:
@Amy1612 - Is this import used? It looks like it might have been accidentally added?
```suggestion

```